static/src/javascripts/projects/common/modules/commercial/reader-revenue-dev-utils.js

```diff
@@ -20,7 +20,6 @@ import {
 import { setGeolocation, getSync as geolocationGetSync } from 'lib/geolocation';
 import config from 'lib/config';
 import { clearParticipations } from 'common/modules/experiments/ab-local-storage';
-import { COOKIE_NAME as CONSENT_COOKIE_NAME } from 'commercial/modules/cmp/cmp-env';
```

Comment:
I believe this is only related to the old CMP.

cc. @guardian/reader-revenue

Reply:
👍